packages/manager/apps/dedicated/client/app/components/user/user.service.js

```diff
@@ -29,18 +30,20 @@ angular.module('services').service('User', [
               userPromiseRunning = false;
 
               if (result) {
-                user = {
-                  ...result.me,
-                  firstName: result.me.firstname,
-                  lastName: result.me.name,
-                  billingCountry: result.me.country,
-                  customerCode: result.me.customerCode,
-                  isEnterprise: indexOf(result.certificates, 'enterprise') > -1,
-                  isVATNeeded:
-                    ['CA', 'QC', 'WE', 'WS'].indexOf(
-                      result.me.ovhSubsidiary,
-                    ) === -1,
-                };
+                user = new User(
+                  {
+                    ...result.me,
+                    firstName: result.me.firstname,
+                    lastName: result.me.name,
+                    billingCountry: result.me.country,
+                    customerCode: result.me.customerCode,
+                    isVATNeeded:
```

Comment:
IMO, if we have a `User` model, `isVATNeeded` could be a method of `User` as same conditions will always applies. What do you think ?

Reply:
@cbourgois I agree but I didn't spend much time on checking and refactoring the usage of `User`. Is that ok with you if we handle it in another PR ?

Reply:
It's ok ;)